admin/check/check_php_inc.php

```diff
@@ -50,8 +50,7 @@
 	'pcre',
 	'Reflection',
 	'session',
-	'mbstring',
-	'fileinfo'
```

Comment:
Please add a comment to document the fact that _fileinfo_ is checked in check_attachments_inc.php

Reply:
Done